src/chainstate/stacks/db/blocks.rs

```diff
@@ -4113,7 +4113,42 @@ impl StacksChainState {
                     let event = STXEventType::STXMintEvent(STXMintEventData { recipient, amount });
                     events.push(StacksTransactionEvent::STXEvent(event));
                 }
-                Ok((total_minted, events))
+
```

Comment:
Wait, we're _not_ going to report STX-transfers from unlocks?  Isn't this a breaking change to the event interface?

Reply:
EDIT: I see that it's getting reported within the lockup receipt.